device-types/Netgear/M4300-28G-PoEp.yaml

```diff
@@ -0,0 +1,196 @@
+---
+manufacturer: Netgear
+model: M4300-28G-PoE+
+slug: netgear-m4300-28g-poep
+part_number: GSM4328PA
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
```

Comment:
This should be removed, you should only use `subdevice_role` if other devices slot into this one, i.e. `device-bays`

`module-bays` do not count as children